### PR TITLE
mgmt, netapp, update live test

### PR DESCRIPTION
**Pull request by weidongxu-microsoft**
It is good now https://dev.azure.com/azure-sdk/internal/_build/results?buildId=4250170&view=results

# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [x] **The pull request does not introduce [breaking changes]**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md).**

## [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#developer-guide)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#building-and-unit-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/netapp/azure-resourcemanager-netapp/src/test/java/com/azure/resourcemanager/netapp/NetAppFilesManagerTests.java [hunk 1]
```diff
@@ -34,15 +35,16 @@ public void beforeTest() {
...
-            .authenticate(credential, profile);
-
         resourceManager = ResourceManager.configure()
             .withLogOptions(new HttpLogOptions().setLogLevel(HttpLogDetailLevel.BASIC))
             .authenticate(credential, profile)
             .withDefaultSubscription();
 
+        netAppFilesManager = NetAppFilesManager.configure()
+            .withLogOptions(new HttpLogOptions().setLogLevel(HttpLogDetailLevel.BASIC))
+            .withPolicy(new ProviderRegistrationPolicy(resourceManager))
```
Comment (weidongxu-microsoft) [inline sdk/netapp/azure-resourcemanager-netapp/src/test/java/com/azure/resourcemanager/netapp/NetAppFilesManagerTests.java:45]: I think this should work.

But for this RP, it was already registered via the explicit code (before I switched to this).

Comment (weidongxu-microsoft) [reply sdk/netapp/azure-resourcemanager-netapp/src/test/java/com/azure/resourcemanager/netapp/NetAppFilesManagerTests.java:45]: tested on databricks, it works https://dev.azure.com/azure-sdk/internal/_build/results?buildId=4250264&view=results

**Review by XiaofeiCao — APPROVED** [review]

**Pull request merged**